JWBestPracticeApps/Podfile.lock

```diff
@@ -5,16 +5,21 @@ PODS:
   - google-cast-sdk/Assets (2.10.0):
     - google-cast-sdk/Core
   - google-cast-sdk/Core (2.10.0)
-  - JWPlayer-SDK (2.5.4)
+  - JWPlayer-SDK (3.2.3)
 
 DEPENDENCIES:
   - google-cast-sdk (<= 2.10.0.4070)
-  - JWPlayer-SDK (~> 2.0)
+  - JWPlayer-SDK (~> 3.2)
```

Comment:
you may need to run pod update; this should have been automatically changed to 3.0

Reply:
The [Podfile.lock](https://guides.cocoapods.org/using/using-cocoapods.html#what-is-podfilelock) file was changed _by_ pod update, not by me manually. Does this address your comment?

Reply:
right, that' s why i'm saying you need to run pod update in order for this to update automatically.